templates/security/fips.html

```diff
@@ -165,45 +207,36 @@ <h2>Certified packages under FIPS 140-2</h2>
         </tr>
       </tbody>
     </table>
-    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140-2 validated modules</a>
+    <a class="p-button--positive js-invoke-modal" href="/security/certifications/docs/fips">Read more about the FIPS certification</a>
```

Comment:
I think this should be `p-button--neutral` judging by the copy doc as only the `Access the fips...` one is green